test/e2e_env/kubernetes/gateway/utils.go

```diff
@@ -22,7 +22,7 @@ func GatewayAPICRDs(cluster Cluster) error {
 	return k8s.RunKubectlE(
 		cluster.GetTesting(),
 		cluster.GetKubectlOptions(),
-		"apply", "-f", "https://github.com/kubernetes-sigs/gateway-api/releases/download/v0.5.0/experimental-install.yaml")
+		"apply", "-f", "https://github.com/kubernetes-sigs/gateway-api/releases/download/v0.5.1/experimental-install.yaml")
```

Comment:
That's easy to miss. Do you know if maybe we could extract this version programatically?

Reply:
I don't know of a clean way to do it. Maybe just a comment in `go.mod` is enough?

Reply:
Yes it's a good enough to start